index.js

```diff
@@ -138,64 +131,11 @@ Client.prototype.sync = function (callback) {
     return self.setTimeUntilReconcile(null, callback)
   }
 
-// begin: legacy updates...
-  if (self.state.ruleset) {
-    self.state.ruleset.forEach(function (rule) {
-      if (rule.consequent) return
-
-      self.state.updatesStamp = now - 1
-      if (self.options.verboseP) self.state.updatesDate = new Date(self.state.updatesStamp)
-    })
-    delete self.state.ruleset
-  }
-  if (!self.state.ruleset) {
```

Comment:
We need to keep this, because we use this for default exclude. So that we exclude gov sites etc. And this logic needs to stay in bl as well

Reply:
Do you just want to cache this in `state` on BL then? Since we shouldn't be using leveldb anymore as that was largely the point of this work

Reply:
I think that we can have this in memory, because it's really small (like we have it now)

Reply:
implemented in browser-laptop ++